### PR TITLE
SW-1904 Fix end-drying reminder date display on accessions view

### PR DESCRIPTION
**Pull request by karthikbtf**
- show relative time left/passed for end-drying reminder
- update padding on mobile view as per spec, current horizontal padding was too big
- update alignment to be `flex-start` on mobile, `center` on desktop - to match the spec

Terraware App 2022-10-03 14-35-26

Terraware App 2022-10-03 16-01-08

### CODE REVIEW

**Comment by constanzauanini** [comment]
This looks good! Can we align the seeds icon of the state with the state name on mobile?

File: src/components/accession2/view/Accession2View.tsx [hunk 1]
```diff
@@ -482,8 +482,19 @@ export default function Accession2View(props: Accession2ViewProps): JSX.Element
           <Box sx={editableParentProps}>
             <Icon name='notification' className={classes.iconStyle} />
             <Box sx={editableProps} onClick={() => setOpenEndDryingReminderModal(true)}>
-              <Typography paddingLeft={1}>
-                {accession?.dryingEndDate ? strings.END_DRYING_REMINDER_ON : strings.END_DRYING_REMINDER_OFF}
+              <Typography paddingLeft={1} lineHeight={isMobile ? 1.2 : 1.5}>
+                {accession?.dryingEndDate
+                  ? `${strings.END_DRYING_REMINDER} ${moment(accession.dryingEndDate).fromNow()}`
```
Comment (charlie-tf) [inline src/components/accession2/view/Accession2View.tsx:487]: this is cool! out of curiosity, does `moment` inherit its locale from global state somehow?

**Comment by karthikbtf** [comment]
> This looks good! Can we align the seeds icon of the state with the state name on mobile?

Kinda looks like this in the spec (see mobile left hand side).. i do agree with you though. Will try something.

image-20221003-185610

**Comment by karthikbtf** [comment]
> > This looks good! Can we align the seeds icon of the state with the state name on mobile?
> 
> Kinda looks like this in the spec (see mobile left hand side).. i do agree with you though. Will try something.
> 
> image-20221003-185610

Terraware App 2022-10-04 08-23-20

fixed

**Pull request merged**